app/javascript/components/journey/Contribution.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { fromNow } from '../../utils/time'
+import { GraphicalIcon, Reputation } from '../common'
+
+export type ContributionProps = {
+  id: string
+  value: number
+  description: string
+  iconName: string
+  internalLink?: string
+  externalLink?: string
+  awardedAt: string
+  track?: {
+    title: string
+    iconName: string
+  }
+}
+
+export const Contribution = ({
+  value,
+  description,
+  iconName,
+  internalLink,
+  externalLink,
+  awardedAt,
+  track,
+}: ContributionProps): JSX.Element => {
+  const link = internalLink ? internalLink : externalLink
```

Comment:
I'm not entirely sure, but can this also be written as:

```suggestion
  const link = internalLink || externalLink
```

?

Reply:
@kntsoriano I've committed this so I can merge this PR into the other. Sorry if that's wrong :)